src/v2/models/AppState.js

```diff
@@ -49,7 +49,7 @@ export default Model.extend({
       },
     },
     authenticatorMethodType: {
-      deps: ['currentAuthenticator', 'currentAuthenticatorEnrollment',],
+      deps: ['currentAuthenticator', 'currentAuthenticatorEnrollment'],
```

Comment:
Fixing a typo.

Reply:
So, the other similar places don't have a trailing `,` in the same file. It just makes it consistent (doesn't make any difference as you already know)